shared/domain/src/main/java/com/ivy/domain/features/FeatureGroup.kt

```diff
@@ -0,0 +1,5 @@
+package com.ivy.domain.features
+
+enum class FeatureGroup {
+    CATEGORY, ACCOUNT, OTHER
```

Comment:
For enums we follow a more Kotlin idiomatic styling:

Category, Account, Other

Reply:
Done